src/fmu/dataio/providers/objectdata/_base.py

```diff
@@ -260,7 +187,11 @@ def _derive_named_stratigraphy(self) -> DerivedNamedStratigraphy:
         `stratigraphy`. For example, if "TopValysar" is the model name and the actual
         name is "Valysar Top Fm." that latter name will be used.
         """
-        name = derive_name(self.dataio, self.obj)
```

Comment:
I felt this was simple enough to just include in this function, and the name was a bit misleading because it was deriving the name for the name stratigraphy. That ends up being used for the object, too, but was handled after this